components/sidebar/sidebar_channel/sidebar_channel_link/index.ts

```diff
@@ -33,6 +34,12 @@ function makeMapStateToProps() {
 
         const unreadCount = getUnreadCount(state, ownProps.channel.id);
 
+        const getCategory = makeGetCategory();
+        const preferences = getCategory(state, Preferences.AB_TEST_PREFERENCE_VALUE);
+        const firstChannelNameFromPref = preferences.find((pref: PreferenceType) => pref.name === RecommendedNextSteps.CREATE_FIRST_CHANNEL);
+        const firstChannelNameFromRedux = state.views.channelSidebar.firstChannelName;
+        const firstChannelName = firstChannelNameFromRedux || firstChannelNameFromPref?.value;
```

Comment:
because there could be the situation where after the first channel creation, the window is closed or for some reason the flow is disrupted, if we keep the info only in the redux store, it could cause potential problems where the tips would not show. But even when we persist in the preferences too, the preference value is not enough to auto trigger the sidebar_channel link component rerender, so we need both.